pkg/local_object_storage/metabase/metadata.go

```diff
@@ -318,12 +319,10 @@ func (db *DB) searchTx(tx *bbolt.Tx, cnr cid.ID, fs object.SearchFilters, fInt m
 				return nil, nil, fmt.Errorf("invalid primary filter value: %w", err)
 			}
 		} else {
-			prefixByte := metaPrefixAttrIDPlain
-			if intPrimMatcher {
-				prefixByte = metaPrefixAttrIDInt
-			}
+			prefixByte := metaPrefixAttrIDInt
 			primSeekKey = slices.Concat([]byte{prefixByte}, []byte(primAttr), utf8Delimiter)
 			primSeekPrefix = primSeekKey
+			jumpToPlains = !intPrimMatcher
```

Comment:
Is it possible to jump in and out according to ASCII?

Reply:
may u pls clarify ur question, i didnt get what u mean

Reply:
a=+
a=.
<switch to numerics>
a=0
a=90
<switch back to symbols>
a=A

But the more I think about it the more I agree we should keep it simple. In general we're not expecting a lot of type mixes, most users should be careful with their attributes.

Reply:
oh, i thought about this, can be done w/ 2 cursors pretty easily, but can change code huge. Decided this just not worth it for now. But i'll leave an issue